group_role_mappings.go

```diff
@@ -0,0 +1,124 @@
+package keycloak
+
+import (
+	"gopkg.in/h2non/gentleman.v2/plugins/body"
+	"gopkg.in/h2non/gentleman.v2/plugins/url"
+)
+
+const (
+	groupsRealmRoleMappingPath      = "/auth/admin/realms/:realm/groups/:id/role-mappings/realm"
+	groupsRoleMappingByClientIdPath = "/auth/admin/realms/:realm/groups/:group_id/role-mappings/clients/:id"
+)
+
+//GetClientRoleMappings get the realm level roles for the group or error (if any).
+// > GET http://localhost:8000/auth/admin/realms/demo/groups/80206962-5dcb-4252-8cbb-2e828c1d010b/role-mappings/realm
+// ```json
+// [
+//   {
+//     "id": "f815fc8a-5eb6-46c1-a454-5fbc8e1c6492",
+//     "name": "offline_access",
+//     "description": "${role_offline-access}",
+//     "composite": false,
+//     "clientRole": false,
+//     "containerId": "4f0f8206-0ec4-4fd6-99eb-4e8c4b986c43"
+//   }
+// ]
+// ```
+func (c *Client) GetGroupRealmRoleMappings(accessToken string, realmName, groupId string) ([]RoleRepresentation, error) {
```

Comment:
any reason to not remove "string" after accessToken?

Reply:
nope, updated for both eligible methods